vagrant/bootstrap.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# bootstrap script to install ff-node-monitor
+set -x
+
+# The name of your Freifunk community.
+INSTANCE_NAME = "Freifunk Kiel"
```

Comment:
This should have more generic default settings.

Reply:
Actually, ideally the user would specify such things "outside" the main shell file -- so that they can change their settings but still `git pull` to get updates. How is that usually handled with vagrant?

Reply:
i created an extra .dist config file